ios/iden3CoreSDK/Iden3ProveClaimDelegate.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Iden3EventDelegate.swift
+//  iden3CoreSDK
+//
+//  Created by Iden3 on 12/06/2020.
+//  Copyright © 2020 Iden3. All rights reserved.
+//
+
+import Foundation
+
+public protocol Iden3ProveClaimDelegate : NSObjectProtocol {
+    
+    func onClaimProofReceived(proved: Bool)
```

Comment:
I understand that this is used to handle verification response. So what happens here is:
1. The Go code generates a proof
2. The proof is sended to the verifier
3. The verifier replies with a true/false saying if the proof has been verified or it has failed.

With that said, I think that `onClaimProofReceived` is not the best name. It could be something like `onClaimVerification(success: Bool)`

Reply:
done